tests/ImportCommentsPreserved/__snapshots__/ppsi.spec.ts.snap

```diff
@@ -0,0 +1,178 @@
+/* Trailing comment on first line after A (will initially, because this is "close" to last-import 'a' */
+/* Leading comment before B */
+import B from "b";
+
+// Loose trailing comment after imports (should not be dragged up with A)
+
+`;
+
+exports[`import-comments-outer-comments-single-line.ts - typescript-verify > import-comments-outer-comments-single-line.ts 1`] = `
+// Loose leading comment before imports (should not be dragged down with B)
+
+// Leading comment before B
+import B from "b"; // Trailing comment on same-line as B
+// Trailing comment on first line after B (This is treated as a leading comment for next import A!)
+
+// Leading comment before A
+import A from "a"; // Trailing comment on same-line as A
+// Trailing comment on first line after A (will follow A initially, because this is "close" to last-import 'a')
+
+// Loose trailing comment after imports (should not be dragged up with A)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+// Loose leading comment before imports (should not be dragged down with B)
+
+// Trailing comment on first line after B (This is treated as a leading comment for next import A!)
+
+// Leading comment before A
+import A from "a"; // Trailing comment on same-line as A
+// Trailing comment on first line after A (will follow A initially, because this is "close" to last-import 'a')
+// Leading comment before B
+import B from "b"; // Trailing comment on same-line as B
+
+// Loose trailing comment after imports (should not be dragged up with A)
+
+`;
+
+exports[`import-comments-specifier-leading-and-trailing.ts - typescript-verify > import-comments-specifier-leading-and-trailing.ts 1`] = `
+// Top-of-file
+
+import {
+    // j3 leading single-line-comment 1
+    // j3 leading single-line-comment 2
+    j3, // j3 same-line single-line-comment
+    j2, // j2 same-line single-line-comment
+} from "j";
+
+import {
+    /** i3 leading comment
+      */ i3,
+    i2, /* i2 same-line will be treated as leading comment for i1 */
+    i1, /* i1 same-line will be treated as trailing comment for i1 */
+} from "i";
+
+import {
+    h4, // h4 same-line single-line-comment
+    // h3 leading single-line-comment
+    h3, // h3 same-line single-line-comment
+    h2, // h2 same-line single-line-comment
+    h1, // h1 same-line single-line-comment
+} from "h";
+
+// Rest of file below here
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+// Top-of-file
+
+import {
+    h1, // h1 same-line single-line-comment
+    h2, // h2 same-line single-line-comment
+    // h3 leading single-line-comment
+    h3, // h3 same-line single-line-comment
+    h4, // h4 same-line single-line-comment
+} from "h";
+import {
+    i1 /* i1 same-line will be treated as trailing comment for i1 */,
+    i2 /* i2 same-line will be treated as leading comment for i1 */,
```

Comment:
This comment seems worded incorrectly, it's kept as a same-line trailing comment on i2 (which is great!)

Reply:
This comment was added before today’s changes, it no-longer gets force-wrapped.

I’ll fix the wording!